repose-aggregator/components/filters/keystone-v2-filter/src/test/scala/org/openrepose/filters/keystonev2/MockedAkkaServiceClient.scala

```diff
@@ -106,7 +110,10 @@ trait MockedAkkaServiceClient {
                       uri: String,
                       headers: util.Map[String, String],
                       payload: String,
-                      contentMediaType: MediaType): ServiceClientResponse = {
+                      contentMediaType: MediaType,
+                      checkCache: Boolean): ServiceClientResponse = {
+      checkCacheValues += checkCache
+
```

Comment:
This has nothing to do with my changes, but I really don't like that we built our own mock here. I may try and replace all of this with Mockito mocks.